lib/process-services-cloud/src/lib/form/services/content-cloud-node-selector.service.ts

```diff
@@ -54,11 +54,17 @@ export class ContentCloudNodeSelectorService {
   }
 
     async fetchNodeIdFromRelativePath(alias: string, opts: { relativePath: string }): Promise<string> {
-        const nodeEntry: any = await this.apiService.getInstance().node
-            .getNode(alias, opts)
-            .catch((err) => this.handleError(err));
+        const relativePathNodeEntry = await this.fetchNodeId(alias, opts);
+        return relativePathNodeEntry?.entry?.id;
+    }
+
+    async fetchAliasNodeId(alias: string): Promise<string> {
+        const aliasNodeEntry = await this.fetchNodeId(alias);
+        return aliasNodeEntry?.entry?.id;
+    }
```

Comment:
fetchNodeIdFromRelativePath and fetchAliasNodeId are the same methods, the only difference is the opts parameter which you can make optional and keep just one. What do you think?

Reply:
Maybe having has it's benefits as from the names you understand the flow

Reply:
yup, In the first place i had single method but it's little hard to understand the flow. May we can have more generic name so that we can have optional param.

Reply:
I would like to have two method to keep flow understandable. Please do suggest if you have a better name. Thanks